server/src/resolvers/getVariantsResolver/utils/resolveAssembly.ts

```diff
@@ -1,11 +1,6 @@
-const resolveAssembly = (assembly: string) => {
-  if (assembly.includes('37') || assembly.toLowerCase().includes('hg19')) {
-    return '37';
-  } else if (assembly.includes('38')) {
-    return '38';
-  } else {
-    return '';
-  }
-};
+import { AssemblyId } from '../../../types';
+
+const resolveAssembly: (assembly: string) => AssemblyId = assembly =>
+  assembly.includes('38') ? '38' : '37';
```

Comment:
Do you have enforce module boundary typings on for this codebase? If not you might be able to make this less confusing by just typing the argument and adding `as const` after the return.